pages/common/montage.md

```diff
@@ -20,6 +20,6 @@
 
 `montage {{image1.png}} {{image2.jpg}} {{imageN.png}} -geometry +0+0 -tile 2x3 montage_%d.jpg`
 
-- Resize and crop images to completely fill their grid cells before tiling:
+- Resize and crop images to fill their grid cells before tiling:
```

Comment:
This change makes this description potentially ambiguous.

Reply:
Could you elaborate?
I believe the word "completely" should be inferred.

**fill**
_intransitive verb_: To put something into (a container, for example) to capacity or to a desired level.
_intransitive verb_: To supply or provide to the fullest extent.

Reply:
Maybe, but not always in a technical context. For example, does it mean fill as in stretch to fit, or fill as in fill but not changing the aspect ratio?

Reply:
I will concede to the ambiguity you described, but I don't believe the word "completely" has any impact on it.
What you said could be asked regardless of if "completely" is there or not.

Reply:
I think it's fill as in the CSS keyword `cover` here, but I'm unsure as to how to improve the description.